### PR TITLE
Replace run_e3sm template

### PR DESCRIPTION
**Pull request by forsyth2**
Replace run_e3sm template script with most recent version for v2.

[BFB]

### CODE REVIEW

**Comment by forsyth2** [comment]
@rljacob @golaz As discussed earlier, I used the latest `run.v2.LR.piControl.sh` as a template and tested with `XS_2x5_ndays` on Cori-KNL. Are any other tests necessary?

I also added the `[BFB]` tag to the commit since this is just a template change (and therefore presumably won't change any output of the model?)

**Review by rljacob — APPROVED** [review]

**Comment by rljacob** [comment]
Looks good!

**Comment by rljacob** [comment]
@golaz this needs your approval.

**Review by golaz — APPROVED** [review]
Everything looks good to me.

File: run_e3sm.template.sh
Comment (forsyth2) [inline run_e3sm.template.sh:15]: Noticed this typo here. Should I fix before merging?

Comment (rljacob) [reply run_e3sm.template.sh:15]: sure.

Comment (forsyth2) [reply run_e3sm.template.sh]: Fixed.

**Pull request merged**
